openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx

```diff
@@ -149,8 +149,26 @@ const Entitylineage: FunctionComponent<EntityLineageProp> = ({
     status: ElementLoadingState;
   }>(ELEMENT_DELETE_STATE);
 
-  const [nodes, setNodes, onNodesChange] = useNodesState([]);
-  const [edges, setEdges, onEdgesChange] = useEdgesState([]);
+  const { node: initialNode, edge: initialEdge } = useMemo(
+    () =>
+      getLayoutedElementsV1(
+        // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
@ShaileshParmar11 it will be good if we can remove this eslint comment
We can connect to resolve it

Reply:
Had a chat with @ShaileshParmar11 and we have gone through the code. For now we will keep `eslint-disable` comment and in future we will reduce use of `eslint-disable`